diff -urN libmcs-0.6.0/m4/buildsys.m4 SPECS/libmcs-0.6.0/m4/buildsys.m4
--- libmcs-0.6.0/m4/buildsys.m4	2007-10-15 10:27:29.000000000 +0200
+++ SPECS/libmcs-0.6.0/m4/buildsys.m4	2007-12-29 18:10:14.000000000 +0100
@@ -104,7 +104,7 @@
 			AC_MSG_RESULT(POSIX)
 			LIB_CPPFLAGS='-DPIC'
 			LIB_CFLAGS='-fPIC'
-			LIB_LDFLAGS='-shared -fPIC -Wl,-soname=${LIB}.${LIB_MAJOR}.${LIB_MINOR}.0'
+			LIB_LDFLAGS='-shared -fPIC -Wl,-soname=${LIB}.${LIB_MAJOR}'
 			LIB_PREFIX='lib'
 			LIB_SUFFIX='.so'
 			PLUGIN_CPPFLAGS='-DPIC'
